src/lib/customPropTypes.tsx

```diff
@@ -39,27 +37,27 @@ export const suggest = (suggestions: string[]) => {
   const findBestSuggestions = _.memoize((str: string) => {
     const propValueWords = str.split(' ')
 
-    return _.flow(
-      _.map((suggestion: string) => {
+    return _.chain(suggestions)
```

Comment:
Heads up that `_.chain` causes _all_  (or almost all) of lodash to be imported because all the chainable functions must be available on the returned object, and lodash can't statically detect which will be used (i.e. tree shaking will not work). Recommend refactoring to reduce the weight of our library.

https://medium.com/making-internets/why-using-chain-is-a-mistake-9bc1f80d51ba